css/css-view-transitions/only-child-on-root-element-with-view-transition.html

```diff
@@ -36,7 +36,7 @@
     let transition = document.startViewTransition();
     transition.ready.then(() => {
       let style = getComputedStyle(
-        document.documentElement, ":view-transition");
+        document.documentElement, "::view-transition");
```

Comment:
We should have a test that this is invalid, I know @karlcow had created a test about getComputedStyle + pseudo elements, so it could be added there.

Reply:
https://github.com/web-platform-tests/wpt/blob/ee42dc176d02bc8b6bc8d583ed9fb2495db42a4b/css/cssom/getComputedStyle-pseudo.html#L161-L175

Reply:
Yeah, I also pointed to this above. I think that's okay for now, but we could add more coverage.

Reply:
We could change that test to loop over different pseudo elements

Reply:
we could probably cover all of them. That would make sure nothing bad is happening. With the loop being outside of the test. So we know which pseudo-element fails instead of failing everything if one is missing.